src/Components/BankDebitForm/BankDebitForm.tsx

```diff
@@ -13,28 +11,29 @@ import {
 } from "@artsy/palette"
 import { useSystemContext } from "System"
 import { useTracking } from "react-tracking"
-import { LoadingArea } from "../LoadingArea"
 import { InsufficientFundsError } from "Apps/Order/Components/InsufficientFundsError"
 import { preventHardReload } from "Apps/Order/OrderApp"
+import { SaveAndContinueButton } from "Apps/Order/Components/SaveAndContinueButton"
+import { getENV } from "Utils/getENV"
 
 interface Props {
   order: { mode: string | null; internalID: string }
-  returnURL: string
```

Comment:
we don't need to pass returnURL as a prop, it can be defined and used in this component